src/solver/conv_asm_implicit_gemm_gtc_fwd.cpp

```diff
@@ -1502,6 +1504,11 @@ bool ConvAsmImplicitGemmGTCDynamicFwdXdlops::IsApplicable(const ConvolutionConte
     if(miopen::IsDisabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS{}))
         return false;
 
+#if WORKAROUND_SWDEV_306318
+    if(!miopen::IsEnabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS{}))
```

Comment:
If the issue happens with FP32 only, then please limit the W/A to floats. Please also add [FP32] to the title of this PR. Otherwise ignore this comment.

Reply:
I checked fp16 datatype by integer init. It also produce a wrong result. So fp16 is also removed.

Reply:
Good.